Podcast/PlayerRateSlider.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  PlayerRateSlider.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 3/27/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import StepSlider
+
+class PlayerRateSlider: StepSlider {
+
+    var values = PlayerRate.values()
+
+    init() {
+        super.init(frame: .zero)
+        trackHeight = 3
+        trackCircleRadius = 4
+        sliderCircleRadius = 10
+        isDotsInteractionEnabled = true
+        trackColor = .paleGrey
+        sliderCircleColor = .paleGrey
```

Comment:
can you make the big circle color `sea`? just so it's easier to recognize which value you're currently on

Reply:
yup good idea!